### PR TITLE
Add unit tests for `bindService()`

### PR DESCRIPTION
**Pull request by bemasc**
`bindService()` is a function exported by `manager_service` to bind the
service onto a Restify server.  This adds unit tests for this function.

### CODE REVIEW

File: src/shadowbox/server/manager_service.spec.ts [hunk 1]
```diff
@@ -738,6 +740,79 @@ describe('ShadowsocksManagerService', () => {
   });
 });
 
+describe('bindService', () => {
+  let server: restify.Server;
+  let service: ShadowsocksManagerService;
+  let url: URL;
+  const prefix = '/TestApiPrefix';
```
Comment (fortuna) [inline src/shadowbox/server/manager_service.spec.ts:747]: Capital letters

File: src/shadowbox/server/manager_service.spec.ts [hunk 2]
```diff
@@ -738,6 +740,79 @@ describe('ShadowsocksManagerService', () => {
...
+
+    expect(body).toEqual(fakeResponse);
+    expect(service.removeAccessKeyDataLimit).toHaveBeenCalled();
+  });
+
+  // Verify that we have consistent 404 behavior for all inputs.
+  [
+    '/',
+    '/TestApiPre',
+    '/foo',
```
Comment (fortuna) [inline src/shadowbox/server/manager_service.spec.ts:794]: Perhaps add `/TestApiPrefixxx` to make sure it's an exact match?

**Review by fortuna — APPROVED** [review]

File: src/shadowbox/server/manager_service.spec.ts (hunk 2 above)
Comment (bemasc) [reply src/shadowbox/server/manager_service.spec.ts:794]: Done

File: src/shadowbox/server/manager_service.spec.ts (hunk 1 above)
Comment (bemasc) [reply src/shadowbox/server/manager_service.spec.ts]: Done

**Pull request merged**